lib/ansible/modules/cloud/vmware/vmware_dvs_portgroup.py

```diff
@@ -35,6 +35,11 @@
         description:
             - The name of the portgroup that is to be created or deleted.
         required: True
+    portgroup_description:
```

Comment:
Can you change this to just `description` and an alias of `portgroup_description` if you want.